rest.rst

```diff
@@ -285,68 +284,6 @@ Selecting Devices to Test On
 
 To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``. Or you can also set the devices the moment you create a testrun using the `Create a new testrun`_ endpoint.
 
-Start an existing testrun
```

Comment:
Who knows what it can do :)? Let's have it in https://www.pivotaltracker.com/story/show/95058434